IFC4x3/Properties/m/MaximumWindSpeed_1btB4PB6T4pQcpyRRarqLn/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="MaximumWindSpeed_1btB4PB6T4pQcpyRRarqLn" Name="MaximumWindSpeed" UniqueId="65dcb119-2c67-44cd-a9b3-f1b6e4d74571" PrimaryDataType="IfcLinearVelocityMeasure" />
```

Comment:
Is this different than `WindLoadRating`?

Reply:
wind load rating is usually a national class structure based on tests so is an IfcLabel, how is this condition limit measured and reported for railways?

Reply:
The requirement is the wind speed, measured in m/s.

Reply:
thank you! thats all i wanted to know!